providers/salesforce/bulk.go

```diff
@@ -357,35 +289,18 @@ func getColumnIndice(record []string, columnNames []string) (map[string]int, err
 	return indices, nil
 }
 
-func (c *Connector) getIncompleteJobResults(jobInfo *GetJobInfoResult) *JobResults {
-	jobResult := &JobResults{
-		JobId:   jobInfo.Id,
-		State:   jobInfo.State,
-		JobInfo: jobInfo,
-	}
-
+func getIncompleteJobMessage(jobInfo *GetJobInfoResult) string {
```

Comment:
What was the reason behind this refactor? This jobInfo state can stay as the connector method since. Though I see that there is no reference to `c`. Seems more cosmetic I guess. no big deal but just curious.

Reply:
Yes that is cosmetic. I was hesitant. Because naming suggested that it had related sibling method but using `c`, while this one wasn't.
I decided that it is not a connector method. Connector already has a lot of methods.